gallery/tutorials/tutorials/weighted_volume_estimation.py

```diff
@@ -0,0 +1,102 @@
+"""
+Weighted Volume Reconstruction
+==============================
+
+This tutorial demonstrates a weighted volume reconstruction,
+using a published reference dataset.
+"""
+
+# %%
+# Download an Example Dataset
+# ---------------------------
+# ASPIRE's downloader will download, cache,
+# and unpack the reference dataset.
+# More information about the dataset can be found on
+# `Zenodo <https://zenodo.org/records/8186548>`_
+# and in this `paper <https://iopscience.iop.org/article/10.1088/1361-6420/ab4f55/ampdf>`_
+
+from aspire import downloader
+
+sim_data = downloader.simulated_channelspin()
+
+# This data contains a `Volume` stack, an `Image` stack, weights and
+# corresponding parameters that were used to derive images
+# from the volumes.  For example. the rotations below are the known
+# true simulation projections. In practice these would be derived from
+# an orientation estimation component.
+
+imgs = sim_data["images"]  # Simulated image stack (`Image` object)
+rots = sim_data["rots"]  # True projection rotations (`Rotation` object)
+weights = sim_data["weights"]  # Volume weights (`Numpy` array)
+vols = sim_data["vols"]  # True reference volumes (`Volume` object)
+
+# %%
+# Create a ``ImageSource``
+# ------------------------
+# The image stack and projection rotation (Euler) angles can be
+# associated together during instantiation of an ``ImageSource``.
+# Because this example starts with a dense array of images,
+# an ``ArrayImageSource`` is used.
+
+from aspire.source import ArrayImageSource
+
+src = ArrayImageSource(imgs, angles=rots.angles)
+
+# The images are downsampled for the sake of a quicker tutorial.
+# This line can be commented out to achieve the reference size (54 pixels).
+src = src.downsample(24)
+
+# %%
+# .. note::
+#     This tutorial demonstrates bringing reference data.
+#     It is also possible to just create a ``Simulation`` or use other
+#     ``ImageSource`` objects, so long as the rotations required
+#     for backprojecting are assigned.
+
+# %%
+# Volume Reconstruction
+# ---------------------
+# Performing a weighted volume reconstruction requires defining an
+# appropriate 3D basis and supplying an associated image to volume
+# weight mapping as an array.
+
+from aspire.basis import FFBBasis3D
+from aspire.reconstruction import WeightedVolumesEstimator
+
+# Create a reasonable Basis
+basis = FFBBasis3D(src.L, dtype=src.dtype)
+
+# Setup an estimator to perform the back projections and volume estimation.
+# In this case, the `weights` array comes from the reference data set,
+# and is shaped to map images to volumes.
+print("`weights shape:`", weights.shape)
```

Comment:
Might be nice to print number of images and volumes as well to really hit the point home. We do see number of images when `src` is instantiated, but don't see the number of volumes anywhere.

Reply:
Just realizing that the `src` was instantiated with 54 volumes and we're estimating 16. Maybe we should have a note about that somewhere. (Disregard the above comment)

Reply:
The `src` volumes and estimated spectral volumes aren't really the same thing.  I will add a comment, but its probably fair to expect someone looking up how to call this method to have a reasonable idea what the method is supposed to compute.

Reply:
Alright, let me know if the added details clear it up. Thanks

Reply:
Yes, that's clear. Thanks.